clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -297,9 +299,16 @@ private void updateBlockInStream(long blockId) throws IOException {
       mCurrentBlockInStream.close();
     }
     try {
+      if (mTachyonStorageType.isPromote()) {
+        try {
+          mContext.getTachyonBlockStore().promote(blockId);
+        } catch (IOException ioe) {
+          // Failed to promote
```

Comment:
shouldn't we at least log error or info this? I'm not feeling comfortable to let an `IOE` go silently